tutorials/quickstart.ipynb

```diff
@@ -12,8 +12,6 @@
     "\n",
```

Comment:
<p>Not the most beautiful picture to include in the tutorial, but I don't have an alternative so I'll shut up...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>I definitely agree. At the very least, I was hoping to change the color or shape of the nodes based on the node type, but I couldn't figure out how to do it in NetworkX :(. I tried specifying the colors/shapes as lists or dictionaries without any luck but I could try to figure that out.</p>

Reply:
Yeah fair, let's just keep it in mind for the future